src/main/java/io/fusionauth/http/server/HTTPServerConfiguration.java

```diff
@@ -110,6 +112,14 @@ public String getContextPath() {
     return contextPath;
   }
 
+  /**
+   * @return The HTTP unexpected exception handler for this server or null if a default was not set.
+   */
+  public HTTPUnexpectedExceptionHandler getDefaultExceptionHandler() {
```

Comment:
I don't think this method name is correct. Shouldn't it match the `with` method?